dev/vite.config.ts

```diff
@@ -10,7 +10,41 @@ const packageAliases = {};
 glob.sync(resolve(__dirname, "../packages/*/package.json")).forEach(
     (packageJsonPath) => {
         const pkg = JSON.parse(fs.readFileSync(packageJsonPath, "utf-8"));
-        packageAliases[pkg.name] = join(dirname(packageJsonPath), pkg.source);
+
```

Comment:
This is a bit :( 

We need to build a set of aliases for Vite that mimic what we export from each package so that our cross-repo package dependencies work. When we go to production these are package dependencies like anything else, but within this monorepo we need to alias the package imports to the source files within this repo so that we don't have to rebuild to see changes. 

The `package.json` in each `packages/*` folder is still the source of truth, we're just using that data to build the aliases here.